airflow/include/content.go

```diff
@@ -0,0 +1,36 @@
+package include
+
+import _ "embed"
+
+//go:embed dockerignore
```

Comment:
I think ideally this file logic should sit in the `airflow/airflow.go`. `include` subfolder is only meant to hold files required inside a local project and not any code logic.